src/dbnode/storage/namespace/dynamic.go

```diff
@@ -107,8 +107,8 @@ func newDynamicRegistry(opts DynamicOptions) (Registry, error) {
 	}
 
 	logger := opts.InstrumentOptions().Logger()
-	logger.Info(`waiting for dynamic namespace registry initialization.
-		If this takes a long time, make sure that a namespace is configured`)
+	logger.Info("waiting for dynamic namespace registry initialization, " +
```

Comment:
lol +1

Reply:
Heh, I was reading the logs and like... why is there no timestamp for this log line?  Ohh... it's actually a multiline log line with an unencoded endline hah.